<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManualGeoreferencingDialogBase</class>
 <widget class="QDialog" name="ManualGeoreferencingDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Manual Georeferencing - Tie Point Selection</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_instructions">
     <property name="text">
      <string>Instructions will appear here</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_images">
     <item>
      <widget class="QGroupBox" name="groupBox_target">
       <property name="title">
        <string>Target Image (Unreferenced)</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_target">
        <item>
         <widget class="QLabel" name="label_targetImage">
          <property name="minimumSize">
           <size>
            <width>400</width>
            <height>400</height>
           </size>
          </property>
          <property name="styleSheet">
           <string>border: 1px solid gray;</string>
          </property>
          <property name="text">
           <string>Target Image</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_reference">
       <property name="title">
        <string>Reference Image (WMTS)</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_reference">
        <item>
         <widget class="QLabel" name="label_referenceImage">
          <property name="minimumSize">
           <size>
            <width>400</width>
            <height>400</height>
           </size>
          </property>
          <property name="styleSheet">
           <string>border: 1px solid gray;</string>
          </property>
          <property name="text">
           <string>Reference Image</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_tiePoints">
     <property name="title">
      <string>Tie Points</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_tiePoints">
      <item>
       <widget class="QTableWidget" name="tableWidget_tiePoints">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>150</height>
         </size>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_buttons">
        <item>
         <widget class="QPushButton" name="pushButton_clearPoints">
          <property name="text">
           <string>Clear All Points</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_deleteSelected">
          <property name="text">
           <string>Delete Selected</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget name="pushButton_apply" class="QPushButton">
          <property name="text">
           <string>Apply Tie Points</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_status">
     <property name="text">
      <string>Ready. Click on target image to start selecting tie points.</string>
     </property>
     <property name="styleSheet">
      <string>padding: 5px; background-color: #f0f0f0; border: 1px solid #ccc;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ManualGeoreferencingDialogBase</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ManualGeoreferencingDialogBase</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
